src/EnergyPlus/HVACManager.cc

```diff
@@ -676,47 +644,16 @@ void SimHVAC(EnergyPlusData &state)
     // selectable control schemes based on the logical flags used in
     // this default control algorithm.
 
-    // Using/Aliasing
-    using DataPlant::NumConvergenceHistoryTerms;
-    using EMSManager::ManageEMS;
-    using General::CreateSysTimeIntervalString;
-    using NonZoneEquipmentManager::ManageNonZoneEquipment;
-    using PlantCondLoopOperation::SetupPlantEMSActuators;
-    using PlantManager::GetPlantInput;
-    using PlantManager::GetPlantLoopData;
-    using PlantManager::InitOneTimePlantSizingInfo;
-    using PlantManager::ReInitPlantLoopsAtFirstHVACIteration;
-    using PlantManager::SetupBranchControlTypes;
-    using PlantManager::SetupInitialPlantCallingOrder;
-    using PlantManager::SetupReports;
-    using PlantUtilities::AnyPlantSplitterMixerLacksContinuity;
-    using PlantUtilities::CheckForRunawayPlantTemps;
-    using PlantUtilities::CheckPlantMixerSplitterConsistency;
-    using PlantUtilities::SetAllPlantSimFlagsToValue;
-    using SetPointManager::ManageSetPoints;
-    using SystemAvailabilityManager::ManageSystemAvailability;
-    using ZoneEquipmentManager::ManageZoneEquipment;
-
     // SUBROUTINE PARAMETER DEFINITIONS:
     bool constexpr SimWithPlantFlowUnlocked(false);
     bool constexpr SimWithPlantFlowLocked(true);
 
     // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
     bool FirstHVACIteration; // True when solution technique on first iteration
-    int LoopNum;
-
-    int AirSysNum;
-    int StackDepth;
-    std::string HistoryTrace;
     Real64 SlopeHumRat;
     Real64 SlopeMdot;
     Real64 SlopeTemps;
     Real64 AvgValue;
-    bool FoundOscillationByDuplicate;
-    int ZoneNum;
-    int NodeIndex;
-    bool MonotonicIncreaseFound;
-    bool MonotonicDecreaseFound;
```

Comment:
[src/EnergyPlus/HVACManager.cc:708]:(style),[variableScope],The scope of the variable 'AirSysNum' can be reduced.
[src/EnergyPlus/HVACManager.cc:709]:(style),[variableScope],The scope of the variable 'StackDepth' can be reduced.
[src/EnergyPlus/HVACManager.cc:710]:(style),[variableScope],The scope of the variable 'HistoryTrace' can be reduced.
[src/EnergyPlus/HVACManager.cc:715]:(style),[variableScope],The scope of the variable 'FoundOscillationByDuplicate' can be reduced.
[src/EnergyPlus/HVACManager.cc:716]:(style),[variableScope],The scope of the variable 'ZoneNum' can be reduced.
[src/EnergyPlus/HVACManager.cc:717]:(style),[variableScope],The scope of the variable 'NodeIndex' can be reduced.
[src/EnergyPlus/HVACManager.cc:718]:(style),[variableScope],The scope of the variable 'MonotonicIncreaseFound' can be reduced.
[src/EnergyPlus/HVACManager.cc:719]:(style),[variableScope],The scope of the variable 'MonotonicDecreaseFound' can be reduced.